frontend/src/apis/index.tsx

```diff
@@ -374,6 +417,29 @@ export async function fetchServiceDescriptions(params: FetchServicesParam): Prom
   }
 }
 
+export async function fetchModelDescriptions(params: FetchModelsParam): Promise<Model[]> {
```

Comment:
You can fetch model description with `fetchModel~` API actions, so you don't need to implement another one.
`fetchServiceDescriptions` and `fetchService~` are my bad implementation and should be refactored...